frontend/public/assets/css/kiosk.css

```
@@ -417,48 +417,48 @@ form {
 }
-.image--metadata div {
+.asset--metadata div {
   z-index: 1;
 }
-.image--metadata--date {
+.asset--metadata--date {
   font-size: 1.3rem;
 }
-.image--metadata--exif {
+.asset--metadata--exif {
 }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)